collect_app/src/androidTest/java/org/odk/collect/android/feature/formentry/FormAppearancesTest.kt

```diff
@@ -0,0 +1,93 @@
+package org.odk.collect.android.feature.formentry
+
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import org.junit.Rule
+import org.junit.Test
+import org.junit.rules.RuleChain
+import org.junit.runner.RunWith
+import org.odk.collect.android.support.rules.CollectTestRule
+import org.odk.collect.android.support.rules.TestRuleChain
+import org.odk.collect.strings.R
+
+@RunWith(AndroidJUnit4::class)
+class FormAppearancesTest {
+    private val rule = CollectTestRule()
+
+    @get:Rule
+    val ruleChain: RuleChain = TestRuleChain.chain().around(rule)
+
+    @Test
+    fun searchAppearance_ShouldDisplayWhenSearchAppearanceIsSpecified() {
```

Comment:
I get that this is trying to make sure `search` works everywhere it should work (which makes sense to have in one test), but can we cut it down in any way? I don't think we need to assert on every result at this level right?

Reply:
I spit it into smaller tests you are right the big one didn't make sense.